src/rpm/builder.rs

```diff
@@ -216,13 +175,18 @@ impl RPMBuilder {
         self
     }
 
-    pub fn add_changelog_entry<E, F>(mut self, author: E, entry: F, time: u32) -> Self
+    pub fn add_changelog_entry<A, E>(
+        mut self,
+        author: A,
+        entry: E,
+        time: std::time::SystemTime,
```

Comment:
I don't know about this, Unix timestamps are more stable because they never touch timezones.  If a user stores their time as a unix timestamp (like in a database), now there's an unnecessary round trip.  A vec of `SystemTime` would also require 4x as many bits as a vec of u32.

Reply:
It's only ever stored in the builder, and I prefer clear types over floating integers that are just bound by documentation. I'd prefer to define a trait `IntoUnixTimeOffset` and `struct UnixTimeOffset` and make it a traitbound on the arg and convertible from both, `SystemTime` and `u32`

Reply:
Looking at https://doc.rust-lang.org/std/time/struct.SystemTime.html

The only constructor I can see is `SystemTime::now()`  and I don't see any options for `From<..>`.  That's an issue

Reply:
Considering it ends up as a Vec<u32> anyway, I don't really see the point of storing it as something else